docs/sources/installation/binaries.md

```diff
@@ -80,6 +80,8 @@ exhibit unexpected behaviour.
 
 ## Get the docker binary:
 
+### Latest binary for Linux
```

Comment:
Nice contribution!  Two heads in a row are called stacked heads --- meaning no intervening clarifying topic sentence.  I think all the downloads require setting the bit to execute; wasn't able to test Windows I don't have my Windows machine here.

No need to have a notes  in such short sections. I was able to bump up all the head levels making them more visible by putting the one paragraph up and breaking the stacked head mentioned above... other minor tweaks:

---> Markup here: https://gist.github.com/moxiegirl/b08dd0195f24920ff243

## Get a docker binary

You can download the latest release binary or a specific version. After downloading a binary file, you must set the file's execute bit to run it.

The latest binary is available only for Linux. You can download specific binary
versions for Linux, Mac OS X, and Windows. To get the list of stable release
version numbers from Github, view the `docker/docker` [releases
page](https://github.com/docker/docker/releases). 

### Get the latest binary

To download the latest binary for Linux:

    $ wget https://get.docker.com/builds/Linux/x86_64/docker-latest -O docker
    
To set the file's execute bit:

    $ chmod +x docker

If you have trouble downloading the binary, you can also get the smaller,
compressed release file:
[https://get.docker.com/builds/Linux/x86_64/docker-latest.tgz](https://get.
docker.com/builds/Linux/x86_64/docker-latest.tgz)

### Get a Linux binary

To download a specific version for Linux, use the
following URL syntax:

    https://get.docker.com/builds/Linux/i386/docker-<version>
    
    https://get.docker.com/builds/Linux/x86_64/docker-<version>

For example:

    https://get.docker.com/builds/Linux/i386/docker-1.6.0

    https://get.docker.com/builds/Linux/x86_64/docker-1.6.0

You can also get the MD5 and SHA256 hashes:

    https://get.docker.com/builds/Linux/x86_64/docker-1.6.0.md5

    https://get.docker.com/builds/Linux/x86_64/docker-1.6.0.sha256


### Get an OS X binary

The Mac OS X binary is only a client. You cannot use it to run the `docker`
daemon. To download a specific Max OS X binary, use the following URL pattern:

    https://get.docker.com/builds/Darwin/i386/docker-<version>
    
    https://get.docker.com/builds/Darwin/x86_64/docker-<version>

For example:

    https://get.docker.com/builds/Darwin/i386/docker-1.6.0

    https://get.docker.com/builds/Darwin/x86_64/docker-1.6.0

You can also get the MD5 and SHA256 hashes:

    https://get.docker.com/builds/Darwin/x86_64/docker-1.6.0.md5

    https://get.docker.com/builds/Darwin/x86_64/docker-1.6.0.sha256

###  Get a Windows binary
 
You can only download a Windows client binary for version `1.6.0` onwards.
Moreover, binary is only a client, you cannot use it to run the `docker` daemon.
To download a binary use the following pattern:

    https://get.docker.com/builds/Windows/i386/docker-<version>.exe
    
    https://get.docker.com/builds/Windows/x86_64/docker-<version>.exe

For example:

    https://get.docker.com/builds/Windows/i386/docker-1.6.0.exe

    https://get.docker.com/builds/Windows/x86_64/docker-1.6.0.exe

You can also get the MD5 and SHA256 hashes:

    https://get.docker.com/builds/Windows/x86_64/docker-1.6.0.exe.md5

    https://get.docker.com/builds/Windows/x86_64/docker-1.6.0.exe.sha256